joomla/joomla-cms
# Update default.php in mod_banners

Missing container element for _$headerText_ (optional in banner module), which limits the header text's CSS styling. (While _$footerText_ has a proper container element.)

Pull Request for Issue # .

### Summary of Changes
Added **div** container element around _$headerText_ (optional **Header Text** in banner module).


### Testing Instructions
Nothing serious, just select 'HTML5' or 'none' style for module output, then check the output code of the page.


### Actual result BEFORE applying this Pull Request
```
<div class="bannergroup">
  Banner Header Text
 <div class="banneritem">
 ...
```
### Expected result AFTER applying this Pull Request
```
<div class="bannergroup">
  <div class="bannerheader">
    Banner Header Text
  </div>
 <div class="banneritem">
 ...
```
### Documentation Changes Required
Add **bannerheader** class name and its description to this document (or to its newer variant, if exist - couldn't find one though): https://docs.joomla.org/CSS_Class_Names_Frontend_in_Version_1.6
